Source/Scene/Model.js

```diff
@@ -2733,10 +2832,11 @@ define([
             }
         } else {
             createBuffers(model, context); // using glTF bufferViews
-            createPrograms(model, context);
             createSamplers(model, context);
             loadTexturesFromBufferViews(model);
             createTextures(model, context);
+
+            createPrograms(model, context);
```

Comment:
Did you move this because of a dependency?  This should have a comment since it is implicit from the perspective of this function.  We might also need to discuss in person to make sure we are taking the right approach.